translations/texts/objects/apex/apexarmchair/apexarmchair.object.json

```
@@ -44,7 +44,7 @@
     },
     "Texts": {
       "Eng": "Sssickening.",
-      "Rus": ""
+      "Rus": "Тошно."
```

```suggestion
      "Rus": "Отвратительно."
```
Тошно в русском языке подразумевает обычно что говорящему тошно (или скучно), а тут речь идёт об отношении говорящего к предмету